css/localgov-alert-banner.css

```diff
@@ -1,7 +1,44 @@
+/**
+ * @file
+ * Style rules for alert banners
+ */
+
+/* Default */
 .localgov_alert_banner {
+  background-color: #00856A;
+}
+.localgov_alert_banner,
+.localgov_alert_banner a {
+  color: #fefefe;
+}
+.localgov_alert_banner button {
+  color: #121212;
+}
```

Comment:
Is this to stop the button appearing as white with the localgov theme?
That might be better as an adjustment there rather than in module?
(Just a thought, this is fine for now).

Reply:
It's to override the default color value (#fefefe) which would otherwise cascade to the button.

This could be solved by setting a default with a light background and dark color, so the explicit button color wouldn't be needed, or not setting a default at all, since the base alert banner must have a type?

Reply:
@danchamp Do you think
````css
.localgov_alert_banner button {
  color: initial;
}
````
Is widely supported? This sets it back to the system style sheet. Just feels a bit out of place setting a colour for the button.

Reply:
Not IE11 which may be a deal breaker (still widly deployed in councils, though Chrome is default)
https://caniuse.com/#feat=css-initial-value

Reply:
I agree what's there is somewhat clumsy, `initial` would be a neat solution.

Not sure about what level of support we should be giving to IE11, maybe something to be discussed with @finnlewis if it's not already defined?

Reply:
Just found the GDS browser support list:

https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices

Which does include IE11.